examples/raytrace-parallel/index.html

```diff
@@ -0,0 +1,245 @@
+              "green": 1.0,
+              "blue": 1.0
+            }
+          },
+          "albedo": 0.18,
+          "surface": {
+            "Refractive": {
+              "index": 1.5,
+              "transparency": 1.0
+            }
+          }
+        }
+      }
+    },
+    {
+      "Plane": {
+        "origin": {
+          "x": 0.0,
+          "y": -2.0,
+          "z": -5.0
+        },
+        "normal": {
+          "x": 0.0,
+          "y": -1.0,
+          "z": 0.0
+        },
+        "material": {
+          "coloration": {
+            "Color": {
+              "red": 1.0,
+              "green": 1.0,
+              "blue": 1.0
+            }
+          },
+          "albedo": 0.18,
+          "surface": {
+            "Reflective": {
+              "reflectivity": 0.5
+            }
+          }
+        }
+      }
+    },
+    {
+      "Plane": {
+        "origin": {
+          "x": 0.0,
+          "y": 0.0,
+          "z": -20.0
+        },
+        "normal": {
+          "x": 0.0,
+          "y": 0.0,
+          "z": -1.0
+        },
+        "material": {
+          "coloration": {
+            "Color": {
+              "red": 0.2,
+              "green": 0.3,
+              "blue": 1.0
+            }
+          },
+          "albedo": 0.38,
+          "surface": "Diffuse"
+        }
+      }
+    }
+  ],
+  "lights": [
+    {
+      "Spherical": {
+        "position": {
+          "x": -2.0,
+          "y": 10.0,
+          "z": -3.0
+        },
+        "color": {
+          "red": 0.3,
+          "green": 0.8,
+          "blue": 0.3
+        },
+        "intensity": 10000.0
+      }
+    },
+    {
+      "Spherical": {
+        "position": {
+          "x": 0.25,
+          "y": 0.0,
+          "z": -2.0
+        },
+        "color": {
+          "red": 0.8,
+          "green": 0.3,
+          "blue": 0.3
+        },
+        "intensity": 250.0
+      }
+    },
+    {
+      "Directional": {
+        "direction": {
+          "x": 0.0,
+          "y": 0.0,
+          "z": -1.0
+        },
+        "color": {
+          "red": 1.0,
+          "green": 1.0,
+          "blue": 1.0
+        },
+        "intensity": 0.0
+      }
+    }
+  ]
+}
+    </textarea>
+
+    <button disabled id='render'>Loading wasm...</button>
+    <div class='concurrency'>
+      <p id='concurrency-amt'>Concurrency: 1</p>
+      <br/>
+      <input disabled type="range" id="concurrency" min="0" max="1" />
+    </div>
+    <div id='timing'>
+      Render duration:
+      <p id='timing-val'></p>
+    </div>
+
+
+    <canvas id='canvas'></canvas>
+
+    <script>
+      function run() {
+        if (typeof WebAssembly === 'undefined') {
+          alert('WebAssembly is not available!');
+          return
+        }
+        if (typeof SharedArrayBuffer === 'undefined') {
+          alert('SharedArrayBuffer is not available, is the preference not enabled?');
+          return
+        }
+        delete WebAssembly.instantiateStreaming;
```

Comment:
Why do you need this?

Reply:
Oh this is [copied from here](https://github.com/rustwasm/wasm-bindgen/blob/ab8be88e7b5db590e17051317d742deb877631fb/examples/no_modules/index.html#L7-L15), tl;dr; the python built-in webservers don't know about the wasm content tyep